lambdas/build-client-oauth-response/src/test/java/uk/gov/di/ipv/core/buildclientoauthresponse/BuildClientOauthResponseHandlerTest.java

```diff
@@ -47,8 +48,11 @@
 import java.util.Map;
 import java.util.stream.Stream;
 
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Can we can avoid adding the `hamcrest` library by using `org.junit.jupiter.api.Assertions`?